mpt/impl/imap-mailbox/external-james/src/test/resources/org/apache/james/imap/scripts/CheckFetch.test

```diff
@@ -0,0 +1,53 @@
+################################################################
+# Licensed to the Apache Software Foundation (ASF) under one   #
+# or more contributor license agreements.  See the NOTICE file #
+# distributed with this work for additional information        #
+# regarding copyright ownership.  The ASF licenses this file   #
+# to you under the Apache License, Version 2.0 (the            #
+# "License"); you may not use this file except in compliance   #
+# with the License.  You may obtain a copy of the License at   #
+#                                                              #
+#   http://www.apache.org/licenses/LICENSE-2.0                 #
+#                                                              #
+# Unless required by applicable law or agreed to in writing,   #
+# software distributed under the License is distributed on an  #
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       #
+# KIND, either express or implied.  See the License for the    #
+# specific language governing permissions and limitations      #
+# under the License.                                           #
+################################################################
+
+S: \* OK IMAP4rev1 Server ready
+
+C: a0000 LOGIN imapuser@domain password
+S: a0000 OK LOGIN completed\.
+
+C: 10 CREATE fetchTest
+S: 10 OK CREATE completed\.
+
+C: A002 SELECT fetchTest
+S: \* FLAGS \(\\Answered \\Deleted \\Draft \\Flagged \\Seen\)
+S: \* \d+ EXISTS
+S: \* \d+ RECENT
+S: \* OK \[UIDVALIDITY \d+\].*
+S: \* OK \[PERMANENTFLAGS \(\\Answered \\Deleted \\Draft \\Flagged \\\Seen( \\\*)?\)\].*
+S: \* OK \[HIGHESTMODSEQ \d+\].*
+S: \* OK \[UIDNEXT 1\].*
+S: A002 OK \[READ-WRITE\] SELECT completed\.
+
+C: A003 APPEND fetchTest (\Flagged) {93+}
+C: From: expeditor@example.com
+C: To: recipient@example.com
+C: Subject: test subject
+C:
+C: Test content
+S: \* 1 EXISTS
+S: \* 1 RECENT
+S: A003 OK (\[.+\] )?APPEND completed\.
+
+C: A10 FETCH *:* (BODY UID)
+S: \* 1 FETCH \(BODY \("TEXT" "PLAIN" \("charset" "us-ascii"\) NIL NIL "7BIT" 12 0\) UID 1\)
+S: A10 OK FETCH completed\.
+
+C: 20 DELETE fetchTest
```

Comment:
could you use consistent tags?

Reply:
10?

Reply:
A11

Reply:
thanks

Reply:
Actually, I didn't repeat myself, but all tags should be consistent, which is not the case.